_news/2018-11-27-visualize-gcp.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Visualize your GCP Architecture using Forseti
+author: Mike Zinni
+---
+Customers often struggle to create hand-wired diagrams of their cloud app’s
+architecture, and keeping those diagrams up-to-date as things change over time
+isn’t fun either…
+
+But now, by combining [Forseti Security’s](https://forsetisecurity.org/) 
+Inventory Scanner, some clever-but-fairly-gnarly Cloud SQL, and
+[D3.js](https://d3js.org/) 
+collapsible trees, you can automatically generate interactive architecture
+diagrams that represent your GCP environment, much like this one…
```

Comment:
I think that it would be awesome to add a snapshot image of the diagram here.

Reply:
Awesome, I can see the image, didn't know that the image can be pulled externally like this.